docs/package.json

```diff
@@ -18,7 +18,7 @@
     "@mdx-js/react": "^1.6.18",
     "@moxy/next-compile-node-modules": "^2.1.1",
     "@xstate/inspect": "^0.4.1",
-    "amplify-docs": "github:https://github.com/aws-amplify/docs#33d383d",
+    "amplify-docs": "github:https://github.com/aws-amplify/docs#481fb50",
```

Comment:
commit `33d383d` only existed on a fork somehwere, so I updated it to `481fb50` which exists on the `next` branch of aws-amplify/docs. See https://github.com/aws-amplify/docs/pull/3485/commits/481fb50a8540e92cf0af824fe8edb2a83815d72f